pkg/client/b19client.go

```diff
@@ -223,22 +235,153 @@ func (mb *b19client) NewAirSpeedControl(data []uint16) (results *protocol.Protoc
 	return resp, nil
 }
 
+// FloorHeatingOn sends the ON command for Floor Heating unit to gateway
+func (mb *b19client) FloorHeatingOn(data []uint16) (results *protocol.ProtocolDataUnit, err error) {
```

Comment:
the only difference between this function and the ) FloorHeatingOff is the newArr. Should be able to reduce the code duplication by change both function and extract the reusable logic to a new function by passing in different newArr value